docs/tutorials/nuvlabox/gpu-demo.md

```diff
@@ -1,73 +1,99 @@
 ---
 layout: page
-title: GPU Application Demo
+title: GPU App at the Edge
 nav_order: 1
 parent: Tutorials
 has_children: false
 ---
 
-# Demo: AI accelaration using GPU
+# Demo: AI accelaration using GPU, on an NVIDIA Jetson Nano
 
 ## Scenario
 
-The goal of this demo is to demonstrate the GPU capabilities of Docker, and how to use these capbilities through Nuvla.
+The goal of this demo is to demonstrate the GPU capabilities of Docker, and how to use these capabilities through Nuvla.
 
-Since Docker 19, GPU usage when running a container can be achieved using --gpus all - this flag allows Docker to pass the GPU devices to the container.
-This capability is only available to Nvidia GPUS. While the Docker client is able to use the GPU flag, Docker Compose hasn't caught up yet. This means we can't
-use a docker-compose command to pass the GPU. 
+Since Docker 19, GPU usage when running a container can be achieved using `--gpus all` - this flag allows Docker to pass the GPU devices to the container.
+This capability is only available to NVIDIA GPUs. While the Docker client is able to use the GPU flag, Docker Compose hasn't caught up yet. This means we can't natively use a Compose file to make use of the `--gpus` property.
 
-We'll deploy a Docker container through Nuvla. To be able to use the GPU, we need to specify the devices and the libraries that are used by the GPU, and pass them through with Docker-Compose.
+Let's say we have a Compose file-defined application to be deployed from Nuvla. Given the aforementioned Docker limitation, we need to manually select the GPU devices and libraries from the host, and pass them to the container via the Compose file.
 
-To know the correct devices and libraires, you can use the GPU discovery tool installed with NuvlaBox.
+To know the correct devices and libraries, we can use the [GPU discovery tool](https://github.com/nuvlabox/peripheral-manager-gpu) installed with NuvlaBox.
+
+![gpuDemoSchema](/assets/img/gpu-demo-scheme.png){: :center}
```

Comment:
do you have the rights to use that image? Also, it shouldn't say "private app store". This is a tutorial so the app in Nuvla must be made public for the readers to try

Reply:
I made the image. Or are you talking about an asset in the image?

Reply:
the bicycle/car image

Reply:
No, that comes with Darknet. I'll change it

Reply:
after fixing the camera position and having a real stream, pls replace this image